Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -46,6 +50,32 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager) {
         masterTalons = new HashMap<XCANTalon, BaseDriveSubsystem.MotionRegistration>();
         masterTalons.put(leftMaster, new MotionRegistration(0, 1, -1));
         masterTalons.put(rightMaster, new MotionRegistration(0, 1, 1));
+        
+        
+        leftMaster.createTelemetryProperties("LeftDriveMaster");
+        rightMaster.createTelemetryProperties("RightDriveMaster");
+        
+        leftTicksPerFiveFeet = propManager.createPersistentProperty("leftDriveTicksPer5Feet", 0);
+        rightTicksPerFiveFeet = propManager.createPersistentProperty("rightDriveTicksPer5Feet", 0);
+        
+    }
+    
+    public double ticksToInches(double ticks) {
+        double tpi = ticksPerInch();
+        
+        if (tpi != 0) {
+            return (1.0 / ticksPerInch()) * ticks;
```

Comment:
What 1/x * v rather than straight division?

Reply:
Was just keeping the pattern consistent between the ticksToInches and inchesPerTicks, where it's

`
factor * inputArgument
`

But if that makes it unintuitive to read, then it hasn't served its purpose.